packages/jaeger-ui-components/src/TraceTimelineViewer/SpanDetailRow.tsx

```diff
@@ -123,7 +123,7 @@ export class UnthemedSpanDetailRow extends React.PureComponent<SpanDetailRowProp
     const styles = getStyles(theme);
     return (
       <TimelineRow>
-        <TimelineRow.Cell width={columnDivision}>
+        <TimelineRow.Cell width={columnDivision} style={{ overflow: 'hidden' }}>
```

Comment:
A background of this element is colored and as we removed background on the details cell, we need to prevent overflow as the positioning is indented based on the indent of the particular span while width is the same.